ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/UserManagementStore.scala

```diff
@@ -79,16 +79,19 @@ trait UserManagementStore {
       loggingContext: LoggingContext
   ): Future[Result[User]]
 
+  // TODO um-for-hub major: Validate the size of update annotations is within max annotations size
```

Comment:
fyi @mziolekda just now I added these comments - marked with 'TODO um-for-hub major - I plan to handle them in a separate PR, not to hold up this PR for too long